PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -1 +1,269 @@
-(placeholder)
+<!--
+
+This is a template to be used for methodology development. The content of this file shall follow the Markdown conventions with Github flavour.
+
+
+If you need help on the Markdown syntax for Github, here is a good link:
+https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
+
+And if you need a tool to make the rendering work in VSCode, here is a good plugin:
+https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles 
+
+--> 
+# Small scale solar self-consumption methodology
+
+**Name**: *Small scale solar self-consumption*
+
+**Developed by**: *[PeerCo](www.peerco.earth)*
```

Comment:
In the voluntary carbon markets, the people who develop the methodology are paid a small licensing fee of some kind to recover costs of developing the methodology in the first place. In this model, the group that develops and approves this would be compensated. Does the initiating company get some form of 'lead author' or 'initiating company' designation so that any questions can go back to us specifically. Some of the people in the technical committee might be representing their personal views, whereas PeerCo is initiating this as a company. There needs to be a clear distinction or understanding. One to be discussed in the steering committee.